test/test_kms_with_cli.py

```diff
@@ -132,9 +132,13 @@ def no_bool_convert(pairs):
 
 def test_GenerateQuote_and_VerifyQuote(base_url, headers):
     print('====================test_GenerateQuote_and_VerifyQuote start===========================')
+    mrenclave = str(base64.b64encode("00000000000000000000000000000000".encode("utf-8")),'utf-8')
+    mrsigner = str(base64.b64encode("13819519910622110214718767411073544680560000000000000000".encode("utf-8")),'utf-8')
```

Comment:
for the unit test, you can try to add some log in the NAPI_GenerateQuote, after you get the quote, then parse it and get the real mrenclave/mrsigner, and hardcode them in this test case to verify whether it can work,

Reply:
ok